### PR TITLE
Refactor and update the examples to clarify Entry and Premium capabilities

### PR DESCRIPTION
**Pull request by PProfizi**

### CODE REVIEW

**Review by cbellot000 — APPROVED** [review]

File: examples/02-modal-harmonic/00-multi_harmonic.py [hunk 1]
```diff
@@ -3,12 +3,13 @@
...
 Multi-harmonic response example
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
 This example shows how to compute a multi-harmonic response
 using ``fft`` transformations.
 
 .. note::
     This example requires the Premium ServerContext.
-    For more information, see :ref:`_ref_getting_started_contexts`.
+    For more information, see :ref:`_user_guide_server_context`.
```
Comment (anslpa) [inline examples/02-modal-harmonic/00-multi_harmonic.py:12]:
```suggestion
    For more information, see :ref:`user_guide_server_context`.
```

File: examples/02-modal-harmonic/04-modal_superposition.py [hunk 2]
```diff
@@ -12,7 +13,7 @@
 
 .. note::
     This example requires the Premium ServerContext.
-    For more information, see :ref:`_ref_getting_started_contexts`.
+    For more information, see :ref:`_user_guide_server_context`.
```
Comment (anslpa) [inline examples/02-modal-harmonic/04-modal_superposition.py:16]:
```suggestion
    For more information, see :ref:`user_guide_server_context`.
```

File: examples/03-advanced/01-solve_harmonic_problem.py [hunk 3]
```diff
@@ -3,13 +3,14 @@
...
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
 This example shows how to create a harmonic (over frequencies) fields
 container for an analysis with damping. This fields container is then used to
 solve the problem Ma+Dv+Ku=F by inverting the matrix
 
 .. note::
     This example requires the Premium ServerContext.
-    For more information, see :ref:`_ref_getting_started_contexts`.
+    For more information, see :ref:`_user_guide_server_context`.
```
Comment (anslpa) [inline examples/03-advanced/01-solve_harmonic_problem.py:13]:
```suggestion
    For more information, see :ref:`user_guide_server_context`.
```

File: examples/03-advanced/02-volume_averaged_stress.py [hunk 4]
```diff
@@ -11,7 +12,7 @@
 
 .. note::
     This example requires the Premium ServerContext.
-    For more information, see :ref:`_ref_getting_started_contexts`.
+    For more information, see :ref:`_user_guide_server_context`.
```
Comment (anslpa) [inline examples/03-advanced/02-volume_averaged_stress.py:15]:
```suggestion
    For more information, see :ref:`user_guide_server_context`.
```

File: examples/03-advanced/03-exchange_data_between_servers.py [hunk 5]
```diff
@@ -4,14 +4,15 @@
...
+
 In this example, two DPF servers are started, and a workflow is created
 with a part on both servers. This example shows how you can read data
 from a given machine and transform this data on another machine
 without any more difficulties than working on a local computer.
 
 .. note::
     This example requires the Premium ServerContext.
-    For more information, see :ref:`_ref_getting_started_contexts`.
+    For more information, see :ref:`_user_guide_server_context`.
```
Comment (anslpa) [inline examples/03-advanced/03-exchange_data_between_servers.py:15]:
```suggestion
    For more information, see :ref:`user_guide_server_context`.
```

File: examples/02-modal-harmonic/00-multi_harmonic.py (hunk 1 above)
Comment (anslpa) [reply examples/02-modal-harmonic/00-multi_harmonic.py:12]: I think this should be changed everywhere in your PR

**Review by anslpa — APPROVED** [review]

File: examples/03-advanced/02-volume_averaged_stress.py (hunk 4 above)
Comment (PProfizi) [reply examples/03-advanced/02-volume_averaged_stress.py]: Thanks, I was looking at how to fix that, did not guess that one !

File: examples/02-modal-harmonic/00-multi_harmonic.py (hunk 1 above)
Comment (PProfizi) [reply examples/02-modal-harmonic/00-multi_harmonic.py]: Done!

**Pull request merged**